prow/cmd/deck/static/spyglass/spyglass.ts

```diff
@@ -61,13 +71,18 @@ function insertLoading(name: string): void {
 
 window.onload = loadViews;
 
+function showElem(elem: HTMLElement) {
```

Comment:
:robot: `Avoid implicit any by always specifying return types` (this is supposed to be a compile error...)

```suggestion
function showElem(elem: HTMLElement): void {
```